pkg/constant/constant.go

```diff
@@ -11,4 +11,8 @@ const (
 	DefaultServerContextTimeout          = 10 * time.Minute
 	CertificateValidityThreshold float64 = (30.0 / 100.0)
 	RenewalBackoff               float64 = (2.0 / 100.0)
+	SFSSource                            = "sfs"
```

Comment:
We could define these as .proto enums and get it auto generated, and used across all the layers.

Reply:
Done . https://github.com/microsoft/moc/pull/82